CHANGELOG.md

```diff
@@ -1,8 +1,48 @@
+### citus v5.1.0 (May 17, 2016) ###
+
+* Adds support for using EXPLAIN on distributed queries
+
+* Improves planner performance when using tables with many shards
```

Comment:
Maybe some specificity? Improve planner performance for joins with tables involving many shards.